client/src/components/DetailedPatentView.vue

```diff
@@ -152,36 +169,49 @@ export default defineComponent({
             familyAvailable: false,
             index: 0,
             isSubMenuOpen: false,
-            pageDisplay: '',
             noDocuments: false,
-            // Holds the submenu buttons
             optionButtons: [{ iconKey: 'bookmark', action: this.savePatent }],
+            pageDisplay: '',
             patentAvailable: false,
             patentService: new PatentService(),
             saved: true,
         };
     },
     watch: {
+        /**
+         * Watch the extendedPatent reference and load information when it changes
+         * @param newVal    The new value of the extended patent
+         */
         async extendedPatent(newVal: ExtendedPatent): Promise<void> {
             this.patentAvailable = !!newVal;
 
             if (!this.patentAvailable) {
                 return;
             }
 
+            // load documents and family for the patent
             await Promise.all([this.loadDocuments(), this.loadFamily()]);
         },
     },
     created() {
+        // load the patents family when the component is created
```

Comment:
This part of code should be commented as well in the style of /** to be part o generated documentation?

Reply:
Since we didn't comment vue's internal function such as data, created, mounted, ... we should stick with that convention